provisioning/roles/sp_mini_1_create_dirs/tasks/main.yml

```diff
@@ -10,6 +10,24 @@
   shell: 'adduser ubuntu --disabled-password --gecos "" ;
           passwd -d ubuntu'
 
+- name: Insert logrotate configuration for Snowplow Services
+  become: yes
+  copy:
+    dest: "/etc/logrotate.d/snowplow-apps"
+    content: |
+      /var/log/snowplow*.log /var/log/snowplow*.err /var/log/nsq*.log  /var/log/nsq*.err /var/log/iglu_server*.log /var/log/iglu_server*.err {
+          hourly
```

Comment:
the logrotate version by default in the ami doesn't support hourly, and there seems to be both hourly and weekly running at the same time for some reason

Reply:
Adding #147 to this release to have updated logrotate & keep rotating in hourly basis.